apps/sblinky/pkg.yml

```diff
@@ -24,12 +24,8 @@ pkg.author: "Apache Mynewt <dev@mynewt.incubator.apache.org>"
 pkg.homepage: "http://mynewt.apache.org/"
```

Comment:
Generally looks great, thanks for the submission!  Please change the few comments below, and resubmit, and I'll merge it!